iCount/cli.py

```diff
@@ -366,6 +367,14 @@ def main():
     make_parser_from_function(
         iCount.analysis.summary.summary_reports, subparsers)
 
+    # Plotting:
```

Comment:
Why are these commands being registered, if the main rnamaps command is still in the analysis folder?